concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1440,14 +1479,14 @@ jobs:
     plan:
       - aggregate:
           - get: pipeline-trigger
-            passed: [ 'cf-deploy']
+            passed: ['post-deploy','availability-tests']
```

Comment:
Yes, there is small race condition here. The check frequency is every 5 sec. If `post-deploy` can deploy healthcheck app within 5 seconds than avail-test will fail. 

 > why would we not want to run smoke/acceptance/other 

Because we already have one test failure so it doesn't make sense to run others tests?

Why would we like to run more tests if we have failure already? `tag-release` shouldn't be run anyway.  We should fix the issue and re-run entire pipeline. If you want to run `acceptance-tests` after `availability-tests` failure run them manually.